tests/subsys/a_mod/src/common.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _COMMON_H_
```

Comment:
Common is very general, can we name it more accurately?

Reply:
Changed to _COMMON_AUDIO_MODULE_TEST_H_